cristin-project/src/test/java/no/unit/nva/cristin/projects/common/CristinQueryTest.java

```diff
@@ -29,4 +51,30 @@ void buildReturnsUriWithIdAndLanguageWhenIdAndLanguageParametersAreSupplied() {
         URI uri = CristinQuery.fromIdAndLanguage(ID, LANGUAGE_NB);
         assertEquals(GET_ONE_CRISTIN_PROJECT_EXAMPLE_URI, uri.toString());
     }
+
+    @Test
+    void buildReturnsUriWithExtendedListOfParameters() {
```

Comment:
these 3 methods (`buildReturnsUriWithExtendedListOfParameters, buildReturnsUriWithKeyword and buildReturnsUriApproved`) can be combined into one, matching against a uri string with all parameters.

Reply:
Merged tests